lib/UnoCore/Targets/Android/app/src/main/res/values/styles.xml

```diff
@@ -5,8 +5,16 @@
 #if !@(Project.Mobile.ShowStatusbar)
         <item name="android:windowFullscreen">true</item>
 #endif
-        <item name="android:windowIsTranslucent">@(Project.Android.WindowIsTranslucent || 'false')</item>
-        <item name="android:windowBackground">@android:color/transparent</item>
+
+#if @(Project.Android.Splash.Enabled)
+	<item name="android:windowBackground">@drawable/@(Project.Android.Splash.SplashFileName || 'splash_background')</item>
+#else
+	<item name="android:windowFullscreen">true</item>
+	<item name="android:windowContentOverlay">@null</item>
+	<item name="android:windowIsTranslucent">@(Project.Android.WindowIsTranslucent || 'true')</item>
+	<item name="android:windowBackground">@android:color/transparent</item>
```

Comment:
Should indent `<item>`s with 6 spaces to align with other `<item>`s in this file.